.github/workflows/ci.yml

```diff
@@ -78,6 +80,9 @@ jobs:
           toolchain: stable
           components: rust-src
 
+      - name: Run sccache-cache
+        uses: mozilla-actions/sccache-action@v0.0.7
+
       - uses: Swatinem/rust-cache@v2
```

Comment:
Do we need both `sccache-cache` and `rust-cache`?

Reply:
we don't need sccache at all since it doesn't give any benefits in our setup (will write another comment why) - and I'm not sure if we really benefit from rust-cache as is either